File: client/Client/Network/Messages/Session/DenyCreateSessionMessage.cs
namespace LoadRunnerClient.Messages
{
    public class DenyCreateSessionMessage
    {
        public const string TYPE = "DenyCreateSessionMessage";

        public DenyCreateSessionMessage()
        {
        }

        public DenyCreateSessionMessage(string reason)
        {
            _reason = reason;
        }

        private string _reason;

        public string reason
        {
            get => _reason;
            set => _reason = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/DenyCreateSessionMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

public class DenyCreateSessionMessage {

	public static final String TYPE = "DenyCreateSessionMessage";

	private String reason;

	public DenyCreateSessionMessage() {
	}

	public DenyCreateSessionMessage(String reason) {
		this.reason = reason;
	}

	public String getReason() {
		return this.reason;
	}

	public void setReason(String reason) {
		this.reason = reason;
	}
}
